src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -358,5 +359,109 @@ private static int CompareExpandedItem(CompletionItem item1, PatternMatch match1
 
         public static string ConcatNamespace(string? containingNamespace, string name)
             => string.IsNullOrEmpty(containingNamespace) ? name : containingNamespace + "." + name;
+
+        internal static bool TryCreateMatchResult<T>(
+            CompletionHelper completionHelper,
+            CompletionItem item,
+            T editorCompletionItem,
+            string filterText,
+            CompletionTriggerKind initialTriggerKind,
+            CompletionFilterReason filterReason,
+            ImmutableArray<string> recentItems,
```

Comment:
what is recentItems?

Reply:
Recent items are MRU recently committed items

Reply:
#resolved.